enterprise/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/BuildEnterpriseInterpretedExecutionPlan.scala

```diff
@@ -56,7 +56,7 @@ object BuildEnterpriseInterpretedExecutionPlan extends Phase[EnterpriseRuntimeCo
     try {
       val (logicalPlan, pipelines) = rewritePlan(context, from.logicalPlan)
       val idMap = LogicalPlanIdentificationBuilder(logicalPlan)
-      val converters = new ExpressionConverters(CommunityExpressionConverter, EnterpriseExpressionConverters)
+      val converters = new ExpressionConverters(EnterpriseExpressionConverters, CommunityExpressionConverter)
```

Comment:
Before the two converters never took on the same expressions, so the order was not important. With these new specialisations, that is no longer true.